src/wp-includes/cache-compat.php

```diff
@@ -199,3 +199,110 @@ function wp_cache_supports( $feature ) {
 		return false;
 	}
 endif;
+
+if ( ! function_exists( 'wp_cache_get_query_data' ) ) :
+
+	/**
+	 * Retrieves cached query data if valid and unchanged.
+	 *
+	 * @since 6.9.0
+	 *
+	 * @param string $cache_key    The cache key used for storage and retrieval.
+	 * @param string $group        The cache group used for organizing data.
+	 * @param string $last_changed The timestamp of the last modification to the cache group.
```

Comment:
Why is `$last_changed` marked as a string here, even though it's a timestamp?

Reply:
Because sadly `microtime()` without `$as_float = true` is used for these which. I wish we could switch these over, but I think it's a breaking change.

Reply:
Fair point. On a related note, we may want to clarify the doc blocks here though to indicate that this can also be an "identifier" - for example in the scenario where two timestamps are concatenated, in which case it's no longer a timestamp.

Reply:
Good idea.